### PR TITLE
Bring back alignment command

### PR DESCRIPTION
**Pull request by gytis**
https://github.com/snowdrop/snowdrop-automation-client/issues/100

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -20,5 +20,13 @@
         }
       }
     }
+  },
+  "bom": {
```
Comment (metacosm) [inline config/default.json:24]: Why is this needed?

Comment (gytis) [reply config/default.json:24]: You can configure the properties that could be ignored for alignment https://github.com/snowdrop/snowdrop-automation-client/pull/101/files#diff-bb725bdb4483849adf186a500d56c277R11. This was always there.

Comment (metacosm) [reply config/default.json:24]: OK, just wondering why hibernate is ignored by default… and actually, with the current release process I'd argue that, by default, we shouldn't ignore any properties and update all of them.

Comment (gytis) [reply config/default.json:24]: Sure, we could remove all. I think it was there because there was always a big divergence between Hibernate versions in Spring Boot and prod.

Comment (metacosm) [reply config/default.json:24]: Possible, yes but it should be removed by default, I think.

Comment (gytis) [reply config/default.json:24]: I can remove it here, since we've reformatted the bom and changing the way we do the releases.

But just to clarify, I don't think there is a default in this project (I know the config file is called default, but that's just how this lib works). We clone and run the project whenever needed, so all non-private config should be stored in GitHub I think.

**Comment by jacobdotcosta** [comment]
> I've launched the process and only got 3 messages in the log. I don't see any changes made to the sb-2.3.x branch.
> 
> Could you please take a look @gytis ?
> 
> Thanks.
> 
> ```
> 2020-08-05T07:35:49.940Z [m:13:7ec59cda-aa0c-485d-aa90-8c5bd1359b4f:snowdrop:AlignBomWithUpstream:3154] [info ] Setting pom version to 2.3.2-SNAPSHOT
> 2020-08-05T07:35:50.197Z [m:13:7ec59cda-aa0c-485d-aa90-8c5bd1359b4f:snowdrop:AlignBomWithUpstream:3412] [info ] Setting pom parent version to 2.3.2.RELEASE
> 2020-08-05T07:35:50.384Z [m:13:7ec59cda-aa0c-485d-aa90-8c5bd1359b4f:snowdrop:AlignBomWithUpstream:3598] [info ] Setting pom property spring-boot.version=2.3.2.RELEASE
> ```

There were no changes to the versions used that's why the process didn't do anything. Having launched the process against `2.3.0.RELEASE` and the back to `2.3.2.RELEASE` and the changes were applied both to the `pom.xml` file as well as the README.

`2.3.0.RELEASE`: https://github.com/snowdrop/spring-boot-bom/commit/d6a0e5e1584219e8787daa678a1c59c1633cd8e7
`2.3.2.RELEASE`: https://github.com/snowdrop/spring-boot-bom/commit/f51359cf50d0e36f8f46739edff68f723c593749
